contrib/utilities/run_clang_tidy.sh

```diff
@@ -41,9 +41,13 @@ if test ! -d "$SRC/source" -o ! -d "$SRC/include" -o ! -d "$SRC/examples" -o ! -
 fi
 echo "SRC-DIR=$SRC"
 
+# unset CC and CXX
+unset CC
+unset CXX
+
 # enable MPI (to get MPI warnings)
 # export compile commands (so that run-clang-tidy.py works)
-ARGS=("-D" "DEAL_II_WITH_MPI=ON" "-D" "CMAKE_EXPORT_COMPILE_COMMANDS=ON" "-D" "CMAKE_BUILD_TYPE=Debug" "$@")
+ARGS=("-D" "DEAL_II_WITH_MPI=ON" "-D" "CMAKE_BUILD_TYPE=Debug" "$@")
```

Comment:
I have removed `CMAKE_EXPORT_COMPILE_COMMANDS`. We now set this unconditionally :-)